### DISCUSSION TITLE
About adding MSX support

### DISCUSSION THREAD

**Discussion by omubaguette72** [sanni/cartreader#748] [Ideas & Suggestions]
The project already supports many systems and I am very grateful for that.

Recently I found code on the internet to dump MSX roms using Arduino Mega.

Can we use this to add MSX support to OSCR?

Link:
http://ngs.no.coocan.jp/main/wiki.cgi?page=%A5%EC%A5%C8%A5%ED%B4%D8%CF%A2

**Comment by PsyK0p4T** [comment]
this could interest @sakman55 :)

**Reply by sakman55** [reply]
I'll take a look.  Working code is always nice to start off with especially since I don't have an MSX computer.

I'm in the process of acquiring test carts.  I'll be wiring up a prototype adapter soon to begin testing.  If tests go well, then I'll make a PCB adapter.
